### PR TITLE
Space-Travelers-Hub

### PR DESCRIPTION
**Pull request by melashu**
# In this project we have made the following changes 

- We fetched rocket and mission data from the API
- We only fetched data only once when the page loaded the first time 
- We render rockets and missions in the UI. 
- We use the redux toolkit library 
- We used CSS and react-bootstrap 
- We use conditional rendering for reading badges and buttons 
- We filtered out reserved rockets and missions in the profile section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,85 @@
...
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+1. Navigate to a folder where you want the cloned file to appear
+   
+2. Clone this repo in your terminal or git bash using the command
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+  https://github.com/melashu/Space-Travelers-Hub.git
```
Comment (Deepakdanger) [inline README.md:33]: - [ ] I would recommend including the step on how to test the application. Hint : `npm test`

File: src/Store/Store.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import missionReducer from './reducers';
+import rocketReducer from './rocketReducer';
+
+const store = configureStore({
+  reducer: {
+    mission: missionReducer,
+    // dragon: dragonreducer,
+    rockets: rocketReducer,
```
Comment (Deepakdanger) [inline src/Store/Store.jsx:10]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @melashu, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required changes

- [ ] The project should have unit tests with React Testing Library and Jest (for unit tests).

**_Check the comments under the review_**


## Optional changes
You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (melashu) [reply README.md:33]: Thank you @Deepakdanger for your feedback. But testing is not part of the requirement.

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi Team,

Good job guys on making the changes asked by the previous reviewer 👍 
Your project is complete!  

ClappingLeonardoDicaprioGIF (4)

## Highlights:
✅ You used Gitflow correctly
✅ Linter checks passed
✅ README is okay
✅ Good descriptive PR 👍 
✅ App features work as expected and is nicely styled 👍 
✅ Tests were written and all passed 👍 

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**